packages/@lwc/engine-core/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "@lwc/engine-core",
+    "version": "1.7.1",
+    "description": "Core LWC engine APIs.",
+    "homepage": "https://lwc.dev/",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/salesforce/lwc.git",
+        "directory": "packages/@lwc/engine-core"
+    },
+    "bugs": {
+        "url": "https://github.com/salesforce/lwc/issues"
+    },
+    "main": "dist/engine-core.cjs.js",
+    "module": "dist/engine-core.js",
+    "typings": "types/index.d.ts",
+    "license": "MIT",
+    "scripts": {
+        "clean": "rm -rf dist/ types/",
+        "test": "jest",
+        "build": "tsc --emitDeclarationOnly && rollup --config scripts/rollup.config.js"
+    },
+    "files": [
+        "dist/",
+        "types/"
+    ],
+    "devDependencies": {
+        "@lwc/features": "1.7.1",
+        "@lwc/shared": "1.7.1",
+        "@lwc/template-compiler": "1.7.1",
```

Comment:
Sorry I missed asking this in the first pass, but how does engine-core use template-compiler?

Reply:
I think we used to use it when testing internal engine APIs but we no longer do after moving those all to karma integration.